internal/server/data/access_key.go

```diff
@@ -319,5 +323,6 @@ func RemoveExpiredAccessKeys(tx WriteTxn) error {
 	query.B("AND expires_at <= ?", time.Now().UTC().Add(-1*time.Hour)) // leave buffer so keys aren't immediately deleted on expiry.
 
 	_, err := tx.Exec(query.String(), query.Args...)
+	logging.L.Info().Msg("removed expired access key")
```

Comment:
Could do something like this to get a count. I guess this would be pretty useful in general 

```suggestion
	result, err := tx.Exec(query.String(), query.Args...)
	if err != nil {
		return err
	}
	count, err := result.RowsAffected()
	if err != nil {
		return err
	}
	if count > 0 {
		logging.L.Info().Int64("count", count).Msg("removed expired access keys")
	}
```